package.json

```diff
@@ -7,6 +7,7 @@
     "4.0.1"
   ],
   "dependencies": {
-    "hl7.fhir.r4.core": "4.0.1"
+    "hl7.fhir.r4.core": "4.0.1",
+    "hl7.fhir.r5.core": "5.0.0"
```

Comment:
This might cause some issues with diff resolution, are we actually dependent on R5 or just backporting an R5 element into R4?

Reply:
We are dependent on a ValueSet for one of the backports which can only be found within the R5 package. I did wonder about copying the ValueSet on and adding it to the repo, but then it may go out of sync if updated and without us realising

Reply:
The R5-terminology package does not include the `http://hl7.org/fhir/valueset-subscription-payload-content` which is the ValueSet you have included in the backport, only the R5 Core contains this. The only reason this is validating is due to the terminology package having a dependency on `hl7.fhir.uv.extensions.r5` which in turn has a dependency on `hl7.fhir.r5.core":"5.0.0"`. Can we change it back to use the `"hl7.fhir.r5.core": "5.0.0"`  for ease of understanding

Reply:
@ryma2fhir, any impact on diff resolution within simplifier? i.e. will the baseDefinition for our profiles point to the R5 or R4 version of the resource?

Reply:
It shouldn't do as the Profiles and Extensions are explicitly based upon R4 `<fhirVersion value="4.0.1" /`